src/convertToObject.js

```diff
@@ -6,7 +6,48 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrSoursString = sourceString;
```

Comment:
The variable `arrSoursString` is unnecessary since it is just a reference to `sourceString`. You can use `sourceString` directly in your code.